### DISCUSSION TITLE
git.exc.InvalidGitRepositoryError

### DISCUSSION THREAD

**Discussion by mohoray** [gitpython-developers/GitPython#1135] [Q&A] [answered]
Following is the snippet of the code I am using.

```python
import git
repo = git.Repo(search_parent_directories=True)
branch = repo.active_branch
```

After running the code, I have received the following error.

```python
File "/home/user_name/.local/lib/python3.8/site-packages/git/repo/base.py", line 181, in __init__
    raise InvalidGitRepositoryError(epath)
git.exc.InvalidGitRepositoryError: /home/user_name/.local/lib/python3.8/site-packages/git/repo
```

It will be really great if I get some suggestions about solving this error. I am using gitpython version 3.1.12.

Thanks a lot.

**Comment by Byron** [comment, marked as answer]
Does the path at 'current working directory' actually contain a repository? [Here is what GitPython does](https://github.com/gitpython-developers/GitPython/blob/4a1339a3d6751b2e7c125aa3195bdc872d45a887/git/repo/base.py#L140:L181) to figure out if there is a git directory.

Without further information there is not much that can be done here.

**Comment by mohoray** [comment]
I will check that. Thank you
